airflow/ui/src/components/TriggerDag/TriggerDAGModal.tsx

```diff
@@ -94,12 +180,14 @@ const TriggerDAGModal: React.FC<TriggerDAGModalProps> = ({
         <Dialog.CloseTrigger />
 
         <Dialog.Body>
-          <TriggerDAGForm
-            dagParams={dagParams}
-            onClose={onClose}
-            onTrigger={handleTrigger}
-            setDagParams={setDagParams}
-          />
+          {Boolean(error) ? undefined : (
```

Comment:
Why would we want to hide the form if there is an error?

Reply:
This is failure fro dag details page, which eventually won't populate the conf, which might be a problem for users, or mislead them, so thought to hide it and stop from triggering. what do you think?

Reply:
I think it would be better UX to show an error and make the field disabled.

Reply:
Actual this outdated, I display the error in toaster